pkg/server/share_manager.go

```diff
@@ -122,15 +122,15 @@ func (m *ShareManager) Run() error {
 	}
 }
 
-// setupDevice will return a path where the device file can be found
+// SetupDevice will return a path where the device file can be found
 // for encrypted volumes, it will try formatting the volume on first use
 // then open it and expose a crypto device at the returned path
-func setupDevice(logger logrus.FieldLogger, vol volume.Volume, devicePath string) (string, error) {
+func (m *ShareManager) SetupDevice(vol volume.Volume, devicePath string) (string, error) {
```

Comment:
nit: If a method is not exported, lower case should be good enough.

Reply:
Updated. Thank you.